src/Testcontainers.Azurite/AzuriteBuilder.cs

```diff
@@ -66,14 +68,30 @@ public override AzuriteContainer Build()
         return new AzuriteContainer(azuriteBuilder.DockerResourceConfiguration, TestcontainersSettings.Logger);
     }
 
+    public AzuriteBuilder WithAccountCredentials(string accountName, string accountKey)
+    {
+        return Merge(DockerResourceConfiguration, new AzuriteConfiguration(accountName: accountName, accountKey: accountKey))
+            .WithEnvironment("AZURITE_ACCOUNTS", $"{accountName}:{accountKey}");
+    }
+
+    public AzuriteBuilder WithPemCertificate(string certificatePath, string certificateKeyPath)
+    {
+        return Merge(DockerResourceConfiguration, new AzuriteConfiguration(certificatePath: certificatePath, certificateKeyPath: certificateKeyPath))
+            .WithBindMount(Path.GetDirectoryName(GetType().Assembly.Location), "/internal")
```

Comment:
Please do not mount a volume and, instead, utilize the `WithResourceMapping(string, string)` API or one of its overloaded methods to copy the required files to the container before it starts. Mounting a volume is considered an anti-pattern, it does not work properly in all Docker environments.

```csharp
.WithResourceMapping(File.ReadAllBytes(clientCertificateFilePath), ClientCertificateFilePath)
.WithResourceMapping(File.ReadAllBytes(clientCertificateFilePath), ClientCertificateKeyFilePath)
.WithCommand("--cert", ClientCertificateFilePath)
.WithCommand("--key", **ClientCertificateKeyFilePath)
```